packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setPayees.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  CONTRACT_TYPES,
+  ExecuteCommandConfig,
+  ExecutionContext,
+  makeExecuteCommand,
+  getRDD,
+} from '@chainlink/starknet-gauntlet'
+import { ocr2ContractLoader } from '../../lib/contracts'
+
+type PayeeConfig = {
+  transmitter: string
+  payee: string
+}
+
+type UserInput = PayeeConfig[]
+
+type ContractInput = [payees: PayeeConfig[]]
+
+const makeUserInput = async (flags, args): Promise<UserInput> => {
+  if (flags.input) return flags.input as UserInput
+  if ((args.length - 1) % 2 != 0) throw new Error('Invalid number of arguments for payee config')
```

Comment:
I think it's uncommon for this version of gauntlet to pass this data as arguments. Please check how it's done for other similar commands. I'd imagine `--transmitters` and `--payees` flags or something similar.

Reply:
This data is also available in RDD, isn't it?

Reply:
Let's handle RDD input here too, it's a form of user input.

Reply:
thanks, have fixed to use flags instead of arg input.

re RDD: following up on @archseer 's comment here - https://github.com/smartcontractkit/chainlink-starknet/pull/283#discussion_r1210360452, lets decide on a consistent approach here